builds/service-onboarding-build-pack/Jenkinsfile

```diff
@@ -82,10 +87,23 @@ node  {
 						else if (service_config['runtime'].startsWith("go")) {
 							service_template = 'api-template-go'
 						}
+            else if(service_config['runtime'].startsWith("c#")){
+              if(service_config['platform'] == "azure"){
+                service_template = 'azure-api-template-csharp'
+              }
+            }
 						break
 
 					case "function":
-						if (service_config['runtime'].startsWith("nodejs")) {
+            if (service_config['platform'] == "azure") {
+              if (service_config['runtime'].startsWith("nodejs")) {
+                service_template = "function-template-nodejs"
```

Comment:
This can change based one of my previous comments (azure-function-template-nodejs)